### DISCUSSION TITLE
I want to know how to configure the PLL(CCC).

### DISCUSSION THREAD

**Discussion by Usutatsu** [polarfire-soc/polarfire-soc-documentation#254] [Reference designs, FPGA and development kits] [answered]
I want to know how to configure the PLL(CCC).
CCC has 4 unit, CCC_NW, CCC_NN, CCC_SW and CCC_SN.
 
The latest reference designe of ICICLE Kit is using CCC_NW as FPGA fabric clock generator.
However I want to use CCC_SW or CCC_SN as FPGA fabric clock gerator and output fabric clock through GPIO.

Where do I change to set which part of the CCC to use?

_Originally posted by @Usutatsu in https://github.com/polarfire-soc/polarfire-soc-documentation/discussions/251_

**Comment by hughbreslin** [comment, marked as answer]
This is a duplicate of [251](https://github.com/orgs/polarfire-soc/discussions/251), please check there for a response. Someone should be back to you shortly.
